qcodes/dataset/data_set.py

```diff
@@ -857,6 +857,8 @@ def _validate_parameters(*params: Union[str, ParamSpec, _BaseParameter]
                 valid_param_names.append(maybeParam)
         return valid_param_names
 
+    @deprecate('This method has been deprecated.', 'Use `get_parameter_data`'
```

Comment:
I think the reason part "This method has been deprecated" is redundant. The output to user will be like " The function get_data is deprecated, because **This method has been deprecated**." We can only go ahead with giving alternative.

Reply:
Good catch let me rephrase.